pykanidm/kanidm/radius/__init__.py

```diff
@@ -16,12 +16,14 @@
 from . import radiusd
 from .utils import check_vlan
 
+CONTAINER_CONFIG_FILE_PATH = "/data/radius.toml"
+
 # the list of places to try
 CONFIG_PATHS = [
-    os.getenv("KANIDM_RLM_CONFIG", "/data/kanidm"),  # container goodness
-    "~/.config/kanidm",  # for a user
-    "/etc/kanidm/kanidm",  # system-wide
-    "../examples/kanidm",  # test mode
+    os.getenv("KANIDM_RLM_CONFIG", CONTAINER_CONFIG_FILE_PATH),  # container goodness
```

Comment:
For compatability I think we need to leave a default value of "/data/kanidm" here which is what existing deployments will be using (even if we are here to encourage people to use /data/radius.toml).

Reply:
done